capDL-tool/CapDL/Model.hs

```diff
@@ -206,7 +206,7 @@ data KernelObject a
         target :: Word }
 
 -- fake kernel objects for smmu
-    | ARMSID
+    | ARMSID { sidNumber :: Maybe Word }
     | ARMCB { bankNumber :: Maybe Word }
```

Comment:
Depending on what the decision is about stream ids and context banks having their numbers manually specified or automatically generated, it might make sense for these fields to be `Word` instead of `Maybe Word`.

Reply:
I agree it makes sense for a sidNumber to be a word. I guess the difference is not setting the argument would be interpreted as the same as setting it to 0?